package.json

```diff
@@ -22,7 +22,8 @@
     "adb": "adb reverse tcp:9090 tcp:9090 && adb reverse tcp:3000 tcp:3000 && adb reverse tcp:9001 tcp:9001 && adb reverse tcp:8081 tcp:8081",
     "postinstall": "patch-package; node ./bin/postInstall;",
     "build-ios": "react-native bundle --entry-file index.js --platform ios --dev false --bundle-output ios/main.jsbundle --assets-dest ios",
-    "build-android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res"
+    "build-android": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res",
+    "rename-dev-configs": "cp ios/GoogleService-Info.development.plist ios/GoogleService-Info.plist; cp android/app/google-services.development.json android/app/google-services.json"
```

Comment:
I don't think it'll work on Windows machines

Reply:
we can offer that..
![Screenshot 2023-06-14 at 7 44 48](https://github.com/better-rail/app/assets/13344923/97734a4d-36cd-4b59-8a14-d7b4db7a17e2)

but I think it's going too far. let's offer the unix script and add a note for windows user that they should rename the files manually?

Reply:
sounds good